aerotech_101smc2/teststage.py

```diff
@@ -0,0 +1,25 @@
+import serial
```

Comment:
This file can likely be removed entirely, it is specific to a single machine (as it specifies the COM port) and now that we have a working daemon it is superfluous